KubeArmor/core/kubeArmor.go

```diff
@@ -67,6 +67,14 @@ type KubeArmorDaemon struct {
 	HostSecurityPolicies     []tp.HostSecurityPolicy
 	HostSecurityPoliciesLock *sync.RWMutex
 
+	// Audit policies
+	AuditPoliciesMap     map[*tp.K8sKubeArmorAuditPolicy]*tp.KubeArmorAuditPolicy
```

Comment:
wondering why the key and the value are the same types?

Reply:
They were subtly different, but I have changed the structures. Macros now are organized in a map hierarchy. I hope this will make the "macro expansion" code (which I'm working) more simple.